lib/src/commons/BrowserUtils.d.ts

```diff
@@ -228,4 +228,27 @@ export declare namespace BrowserUtils {
      * @param expectedText expected alert's text
      */
     function expectAlertText(expectedText: string): void;
+    /**
+     *
+     * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+     */
+    function mouseButtonDown(mouseButton?: string): void;
+    /**
+     * @param selector - element to move to, If not specified or is null, the offset is relative to current position of the mouse.
+     * @param xOffset - X offset to move to, relative to the top-left corner of the element If not specified, the mouse will move to the middle of the element.
+     * @param yOffset - Y offset to move to, relative to the top-left corner of the element. If not specified, the mouse will move to the middle of the element.
+     */
+    function moveMouseCursorTo(selector?: string, xOffset?: number, yOffset?: number): void;
+    /**
+     * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+     */
+    function mouseButtonUp(mouseButton?: string): void;
```

Comment:
Is it possible to simulate clicking on more than one button simultaneously? If so, the param should probably be an array. If not, then I don't see a reason for a param here.

Reply:
done it is now mandatory. no possibility to click more than one.